### ISSUE TITLE
Merge back agent and firmware 80 to the main repos

### ISSUE THREAD

**Issue by ert78gb** [UltimateHackingKeyboard/agent#2439]
This story collects the main steps of the process.

- [x] 1. Migrate  `master` branch of `agent80` repo
  - revert `scripts/download-firmware.js` to use again the `firmware` repo. The side effect of it Agent will bundle only UHK60 firmware. Because `firmware-uhk80` repo has not been migrated yet and not released a new firmware version that contains the UHK80 specific features. We could do this after step 3, and meanwhile we manually upgrade bundled firmware of the Agent

- [x] 2. Migrate the `master` branch of `firmware-uhk80` repo
  - revet the `lib/agent` git submodule to use the `agent` repo

- [ ] 2.1 make sure that other important branches have been migrated and reopen their PR's
  
- [x] 3. Release a new firmware version

- [x] 4. Release a new Agent version

**Comment by ert78gb** [comment]
I think we could close the issue the main parts already done

**Comment by kareltucek** [comment]
Yes!
